symphony-ws/src/main/java/se/havochvatten/symphony/web/CalculationREST.java

```diff
@@ -512,16 +515,59 @@ public Response multiComparison(@Context HttpServletRequest req, @PathParam("bas
 
         try {
             BaselineVersion baseline = baselineVersionService.getVersionByName(baselineName);
-            CompoundComparisonDto comparison = calcService.createCompoundComparison(request.ids, request.name, principal, baseline);
+            Integer createdCompoundCmpId = calcService.createCompoundComparison(request.ids, request.name, principal, baseline);
+            URI uri = uriInfo.getAbsolutePathBuilder().path(String.valueOf(createdCompoundCmpId)).build();
```

Comment:
Is it really necessary to return the uri with the id instead of only the id?

Reply:
Well, not for the purposes of the frontend application. We do it like this to comply with the [HTTP spec for the 201 CREATED](https://www.rfc-editor.org/rfc/rfc9110#status.201) response. In other words: it's pedantry 🙂

Note that the specified URI is not implemented (yet) as an URL/actual endpoint (as can often be assumed). Here it's intended strictly as an "identifier" for the created resource in the syntax format that the specification demands. A single numeric id (possibly wrapped in a json object) wouldn't be conformant.
If it were of any utility for the frontend app, I'd pass it along in the response body for convenience. At this point the client only checks the response code.